test/viewmodels/NationalInsuranceContributionsViewModelSpec.scala

```diff
@@ -32,16 +34,18 @@ class NationalInsuranceContributionsViewModelSpec extends SpecBase with TableDri
 
   private implicit val messages: Messages = messagesStubbed
 
+  private val nicUrl = nics.routes.Class2NICsController.onPageLoad(taxYear, NormalMode).url
+
   private val testScenarios = Table(
     ("nationalInsuranceStatuses", "userAnswers", "expected"),
     // No statuses, no answers, defaults to cannot start yet until the saving is implemented
-    (Nil, Nil, List(expectedRow("#", NationalInsuranceContributions, CannotStartYet)))
+    (Nil, Nil, List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet)))
```

Comment:
Replace "userAnswers" which isn't being used for "businessStatuses" (or a name similar) for the PR comment I added in NationalInsuranceContributionsViewModel.

Can then replace the comment above for comments and tests for:
- No businessStatuses but test result for each NIC status
- How the various statuses are influenced if adjustments in businessStatuses are different, and if they're different for different businesses

Reply:
Done :)